viewer/api-entry-points/core.ts

```diff
@@ -22,7 +22,12 @@ export {
   ComboControlsEventType,
   CameraEventDelegate,
   CameraChangeDelegate,
-  CameraStopDelegate
+  CameraStopDelegate,
+  IFlexibleCameraManager,
+  FlexibleControlsOptions,
+  WheelZoomType,
+  FlexibleControlsType,
+  MouseActionType
```

Comment:
Can we rename `WheelZoomType` and `MouseActionType` to something more specific as well? (e.g. `Flexible-`)

Reply:
Fixed here: f7871668e8410f478eff38a68d95ea4c055a459e